pre-commit

```diff
@@ -99,7 +99,7 @@ check "Web tests" \
     cargo test -p ipa-core --no-default-features --features "cli web-app real-world-infra test-fixture descriptive-gate"
 
 check "Concurrency tests" \
-    cargo test -p ipa-core --release --features shuttle
+    cargo test -p ipa-core --release --features "shuttle multi-threading"
```

Comment:
not sure if this is an accidental change or no - I don't have a preference which test to run as long as we run both on CI

Reply:
This was intentional, trying to make pre-commit closer to what CI is running.